forge-game/src/main/java/forge/game/combat/CombatUtil.java

```diff
@@ -71,12 +71,20 @@
 public class CombatUtil {
 
     public static FCollectionView<GameEntity> getAllPossibleDefenders(final Player playerWhoAttacks) {
+        // Opponents, opposing planeswalkers, and any battle you don't protect
         final FCollection<GameEntity> defenders = new FCollection<>();
         for (final Player defender : playerWhoAttacks.getOpponents()) {
             defenders.add(defender);
             final CardCollection planeswalkers = defender.getPlaneswalkersInPlay();
             defenders.addAll(planeswalkers);
+            for (Card battle : defender.getBattlesInPlay()) {
```

Comment:
We don't know if all battles can be defenders, but we do know that Sieges can be.